code/src/main/java/habit_mode/model/sudoku/SudokuPuzzle.java

```diff
@@ -0,0 +1,190 @@
+package habit_mode.model.sudoku;
+
+import java.util.Stack;
+
+/**
+ * Sudoku puzzle class
+ * 
+ * @author Matthew Thompson
+ * @version spring 2022
+ */
+public class SudokuPuzzle {
+
+    private int[][] numbers;
+    private boolean[][] numberLocks;
+    private int[][] defaultAnswer;
+    private int selectedNumber;
+    private Stack<SudokuMove> moveHistory;
+
+    /**
+     * Default Constructor
+     * @precondition none
+     * @postcondition this.numbers = new int[9][9] && this.numberLocks = new boolean[9][9] && this.defaultAnswer = new int[9][9] && this.selectedNumber = -1 && this.moveHistory = new Stack<SudokuMove>()
+     */
+    public SudokuPuzzle() {
+        this.numbers = new int[9][9];
+        this.numberLocks = new boolean[9][9];
+        this.defaultAnswer = new int[9][9];
+        this.selectedNumber = -1;
+        this.moveHistory = new Stack<SudokuMove>();
+    }
+
+    /**
+     * Gets the numbers
+     * @precondition none
+     * @postcondition none
+     * @return this.numbers
```

Comment:
We can't see what a private field is from the outside. The return should not mention it, and should tell the developer what the return value is in words.